Automattic/wp-calypso
# server: Serve all requests using evergreen

#### Changes proposed in this Pull Request

* Serve 'evergreen' even for unsupported browsers.
* Drop support for adding `forceFallback` to the URL

Note that IE11 users were able to click 'Continue' on the `/browserhappy` page and get a functional Calypso (they were served `fallback`). After this PR they will get a broken site (they will get served `evergreen`).

#### Testing instructions

* Load calypso.live URL in IE11, you should get redirected to `/browserhappy`. Click on 'continue'. Calypso should "load" but everything should be broken with a WSOD. Using the network tools, verify `calypso/evergreen/*` assets are loaded.

Alternative:
* Checkout this branch and start it with `ENABLE_FEATURES=redirect-fallback-browsers/test`. You should get redirected to `/browserhappy`. Click on 'continue'. Using the network tools, verify `calypso/evergreen/*` assets are loaded.
